api/src/zero/module/aou.clj

```diff
@@ -47,13 +38,6 @@
    :variant_rsids_file                   "gs://broad-references-private/hg19/v0/Homo_sapiens_assembly19.haplotype_database.snps.list"}
   )
 
-(def genotype-concordance
-  "Genotype Concordance inputs for arrays."
-  {:control_sample_vcf_file       "gs://broad-gotc-test-storage/arrays/controldata/NA12878.vcf.gz"
-   :control_sample_vcf_index_file "gs://broad-gotc-test-storage/arrays/controldata/NA12878.vcf.gz.tbi"
-   :control_sample_intervals_file "gs://broad-gotc-test-storage/arrays/controldata/NA12878.interval_list"
-   :control_sample_name           "NA12878"})
-
 (def other-inputs
   "Miscellaneous inputs for arrays."
   {:call_rate_threshold              0.98
```

Comment:
Still not sure about `:genotype_concordance_threshold`, @gbggrant could you help check if we could always pass these by default?

Reply:
The genotype_concordance_threshold can be removed - the WDL defaults to 0.98 and it isn't normally set externally.
However the call_rate_threshold is set in the LIMS message and correspondingly in the arrays message.